build.go

```diff
@@ -36,11 +35,10 @@ import (
 var srcFS embed.FS
 
 var (
-	version    = flag.String("version", "", `version number to build; "build" means to build locally; see --source`)
-	spkBuild   = flag.Int("spk-build", 15, `SPK build number; needs to be monotonically increasing regardless of the --version`)
-	dsmVersion = flag.String("dsm-version", "7", `DSM version(s) to build: 6, 7, or "all"`)
-	goarch     = flag.String("goarch", "amd64", `GOARCH to build package(s) for, "all`)
-	compress   = flag.String("compress", "", `compression option: "speed" or "size"; empty means automatic where local builds are fast but big`)
+	dsmVersion    = flag.String("dsm-version", "7", `DSM version(s) to build: 6, 7, or "all"`)
+	goarch        = flag.String("goarch", "amd64", `GOARCH to build package(s) for, "all`)
+	compress      = flag.String("compress", "speed", `compression option: "speed" or "size"; empty means automatic where local builds are fast but big`)
+	packageCenter = flag.Bool("for-package-center", false, `build for the package center`)
 
 	srcDir = flag.String("source", ".", "path to tailscale.com's go.mod directory root, when using --version=build")
```

Comment:
old docs. there is no --version=build now. there is no --version.

It always builds what's in --source.